flyte.config

```diff
@@ -0,0 +1,11 @@
+[sdk]
+workflow_packages=workflows.getting_started
+python_venv=flytekit_venv
+
+[auth]
+assumable_iam_role=arn:aws:iam::12345:role/flyteworkflows-iad
```

Comment:
can we skip this and start using service-account only?

Reply:
yes the only thing is that if the service account doesn't exist it will fail, whereas this does nothing. i'll just delete it for now and users can read more on auth after they've messed around.

Reply:
updated to use service account